tests/scripts/validate_cluster.sh

```diff
@@ -73,7 +73,12 @@ function test_demo_pool {
 
 function test_csi {
   # shellcheck disable=SC2046
-  return $(wait_for_daemon "test $(kubectl -n rook-ceph get pods --field-selector=status.phase=Running|grep -c ^csi-) -eq 4")
+  #return $(wait_for_daemon "test $(kubectl -n rook-ceph get pods --field-selector=status.phase=Running|grep -c ^csi-) -eq 4")
```

Comment:
leftover line

Reply:
thanks, @leseb I forgot to remove this...will create PR for this.